editor/scene/gui/theme_editor_plugin.cpp

```diff
@@ -1531,6 +1604,29 @@ void ThemeItemEditorDialog::_update_edit_item_tree(String p_item_type) {
 		}
 	}
 
+	{ // Audios.
+		names.clear();
+		edited_theme->get_sound_list(p_item_type, &names);
+
+		if (names.size() > 0) {
+			TreeItem *sound_root = edit_items_tree->create_item(root);
+			sound_root->set_metadata(0, Theme::DATA_TYPE_SOUND);
+			sound_root->set_icon(0, get_editor_theme_icon(SNAME("AudioStream")));
+			sound_root->set_text(0, TTR("sounds"));
```

Comment:
```suggestion
			sound_root->set_text(0, TTR("Sounds"));
```
Following the other cases